ci/scripts/integration_spark.sh

```diff
@@ -18,9 +18,13 @@
 # exit on any error
 set -eu
 
+# SC2034 (warning): source_dir appears unused.
+# shellcheck disable=SC2034
 source_dir=${1}
```

Comment:
It seems that the first argument is Spark version: https://github.com/apache/arrow/blob/f1c6619332a280625a2381b03b2fd4a0722db82f/ci/docker/conda-python-spark.dockerfile#L37-L40

Reply:
set `spark_version` instead of `source_dir`.